editoast/src/views/infra/mod.rs

```diff
@@ -293,9 +293,8 @@ async fn clone(
             .execute(&mut conn).map_err(|err| err.into())
         });
         futures.push(Box::pin(model));
-
         let layer_table = object.get_geometry_layer_table();
-        if layer_table.is_some() {
+        if layer_table.is_some_and(|t| t != "osrd_infra_signallayer") {
```

Comment:
You can apply the condition directly to the SQL request instead of duplicating the whole block.

```rust
let sql = if layer_table != "..." { "..." } else { "..." };
... sql_query(sql).bind ...
```